docs/source/models.rst

```diff
@@ -171,14 +180,15 @@ For more information, see :py:mod:`mlflow.h2o`.
 Keras (``keras``)
 ^^^^^^^^^^^^^^^^^
 
-The ``keras`` model flavor enables logging and loading Keras models. The :py:mod:`mlflow.keras`
-module defines :py:func:`save_model() <mlflow.keras.save_model>` and
-:py:func:`log_model() <mlflow.keras.log_model>` functions that you can use to save Keras models
-in MLflow Model format. These functions serialize Keras models as HDF5 files using the Keras
-library's built-in model persistence functions. MLflow Models produced by these functions
-also contain the ``python_function`` flavor, allowing them to be interpreted as generic
-Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. Finally, you can use
-the :py:func:`mlflow.keras.load_model()` function to load MLflow Models with the
+The ``keras`` model flavor enables logging and loading Keras models. It is available in both python
+and R clients. The :py:mod:`mlflow.keras` module defines :py:func:`save_model()<mlflow.keras.save_model>`
+and :py:func:`log_model() <mlflow.keras.log_model>` functions that you can use to save Keras models
+in MLflow Model format in python. Similarly, in R, you can save or log the model using
+:rlang:func:`mlflow_save_model` and :rlang:func:`mlflow_log_model. These functions serialize Keras
+models as HDF5 files using the Keras library's built-in model persistence functions. MLflow Models
+produced by these functions also contain the ``python_function`` flavor, allowing them to be interpreted
+as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. Finally, you
+can use the :py:func:`mlflow.keras.load_model()` function to load MLflow Models with the
```

Comment:
is there an equivalent in R we should mention?

Reply:
Yes, good point.